setup.sh

```diff
@@ -398,15 +398,27 @@ $sudo footloose status -o json > $status
 jk generate -f config.yaml -f $status setup.js
 rm -f $status
 
+GIT_ORIGIN = `git config --get remote.origin.url`
+GIT_BRANCH = `git rev-parse --abbrev-ref HEAD`
```

Comment:
These aren't valid in bash/zsh.  Can you elaborate on how you tested your changes?

Reply:
They are, what isn’t valid about them?